Makefile

```diff
@@ -1,4 +1,4 @@
-IMG=flagd:latest
+IMG ?= flagd:latest
```

Comment:
I don't think this is necessary as it's outside of the target and can be overridden. Is there an issue with this?